src/client/containers/Wizards/Packages/reducer.ts

```diff
@@ -459,7 +459,9 @@ function isFormValid(data: PackageModel, initialData: PackageModel, view: VIEW_T
                     }
                     return false;
                 }) !== -1;
-            })
+            }) ||
+            data.extraFields.map(ef => ef.value!=null?ef.value.toString():ef.value).sort().join('') !==
```

Comment:
Seems like we should include the key as well as the value here. Otherwise, if field A goes from 1->2 and field B goes from 2->1, I think this wouldn't detect it as being changed, correct?

Reply:
True.  That is an edge possibility.  I updated and concated the property name to the values as well.  Which actually cleaned it up a bit not needing to check if value was null since name is required and not needing to do .toString.